docs/api-docs/catalog/v2-vs-v3.md

```diff
@@ -0,0 +1,105 @@
+* Global option set rules are copied as product rules and the `sort_order` is updated so that they are executed before any existing product rules, which should mirror the behavior before the product was changed.
+
+What this does not do:
+
+* Remove the option set from the store entirely. It is still available in the control panel as an option set to be assigned.
+* Change product pricing, rules, or any other product modifiers. They will be copied over and assigned the product correctly.
+
+The following workflows demonstrate the difference between creating products with variants in V2 Products and V3 Catalog APIs.
+
+**Workflow to create product options and option sets in V2 Products:**
+
+1. Create the product
+2. Create the options
+3. Create an option set
+4. Assign the option set to the product
+5. Create adjustments, such as price adjustment, using rules
+
+
+**Workflow to create products with variants in V3 Catalog:**
+
+1. Create the product with variants in one call
+2. Create adjustments, such as price adjustment, directly on the variant or modifier
+
+### Product Option Sets
+
+V3 Catalog does not include an endpoint to manage option sets but it will respect option sets created using V2 Products or the control panel. Currently, the control panel’s Add/Edit Product section consumes the V2 API and any products created and managed through the control panel will be converted to the V2 model using option sets. If you apply an option set to a V3 product, the product’s variants will be removed.
+
+### Product Rules 
+
+Most of the use cases for using V2 product rules can be solved by making adjustments directly on V3 variants and modifier options. For cases where an adjustment depends on the selection of multiple modifier values, use V3’s [complex rules](https://developer.bigcommerce.com/api-reference/store-management/catalog/product-complex-rules/createcomplexrule) resource. 
+In V3, any variants or modifier adjusters created with non-null core properties such as price, weight, image, and purchasability will create a rule on the back-end. Such variants and modifier options will appear in V2 as product rules and their edits will be shared across API versions.
+
+### Product Variants
+
+In V3, every purchasable entity in the catalog is a variant, including the product itself. This enables enhanced inventory management flows such as the ability to use the [variants](https://developer.bigcommerce.com/api-reference/store-management/catalog/variants/updatevariantsbatch) endpoint to manage inventory levels. While it is possible to create a SKU with a subset of product options using the V2 API; in V3, variants must be created for every combination of option values. We recommend creating products using the V3 API as BigCommerce intends to move operations to the V3 API.
+
+To lower the number of API calls being made, variants can be included with a GET request using `?include=variants`.
```

Comment:
To reduce the number of API calls made, include variants with a GET request using `?include=variants`.